.github/workflows/tests.yml

```diff
@@ -21,8 +21,7 @@ jobs:
       uses: shivammathur/setup-php@v2
       with:
         php-version: ${{ matrix.php }}
-        tools: composer
-        coverage: pcov
+        tools: composer:v2
```

Comment:
Wouldn't this be the default?

Reply:
yes, in time I've created this it wasn't.^^

Reply:
The world is a better place now 🙂